### PR TITLE
Add internal/config/lb test

### PR DESCRIPTION
**Pull request by hlts2**
### Description:

I added test for internal/config/lb and godoc comment.

**grammar check passed**

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/config/lb.go [hunk 1]
```diff
@@ -17,6 +17,7 @@
 // Package config providers configuration type and load configuration logic
 package config
 
+// LB represents the configuration for lb gateway.
```
Comment (hlts2) [inline internal/config/lb.go:20]:
```suggestion
// LB represents the configuration for LB.
```

File: internal/config/lb_test.go [hunk 2]
```diff
@@ -53,53 +55,122 @@ func TestLB_Bind(t *testing.T) {
...
+						AgentPort:      8081,
+						AgentName:      "vald-agent-ngt",
+						AgentNamespace: "vald",
+						AgentDNS:       "vald-agent-ngt.vald.svc.cluster.local",
+						NodeName:       "vald-01-worker",
+						IndexReplica:   3,
+					},
+				},
+			}
+		}(),
```
Comment (vankichi) [inline internal/config/lb_test.go:80]: How about define the specific variable before `return test{}` and use them in `test{}` struct?
like as:
```go
func () test {
    agentPort := 8081
    return test {
        ...
        fields: fields{
            AgentPort: agentPort,
            ....
        },
        want: want{
            want: &LB {
                AgentPort: agentPort,
                ....
            },
            ....
        }
    }
}
```
You use the functional test codes, you can use them for more readable and easy to maintenance.

Comment (hlts2) [reply internal/config/lb_test.go:80]: thank you for your review.
I think it looks good to me!
I will fix it soon!

File: internal/config/lb_test.go [hunk 3]
```diff
@@ -53,53 +55,141 @@ func TestLB_Bind(t *testing.T) {
...
+		func() test {
+			agentPort := 8081
+			agentName := "vald-agent-ngt"
+			agentNamespace := "vald"
+			agentDNS := "vald-agent-ngt.vald.svc.cluster.local"
+			nodeName := "vald-01-worker"
+			indexReplica := 3
+
```
Comment (vankichi) [inline internal/config/lb_test.go:95]:
```suggestion
```

File: internal/config/lb_test.go [hunk 4]
```diff
@@ -53,53 +55,141 @@ func TestLB_Bind(t *testing.T) {
...
+					},
+				},
+			}
+		}(),
+		func() test {
+			agentName := "vald-agent-ngt"
+			agentNamespace := "vald"
+			agentDNS := "vald-agent-ngt.vald.svc.cluster.local"
+			nodeName := "vald-01-worker"
+
```
Comment (vankichi) [inline internal/config/lb_test.go:136]:
```suggestion
```

**Review by vankichi — COMMENTED** [review]
LGTM after fix these

**Review by vankichi — DISMISSED** [review]

**Review by kevindiu — DISMISSED** [review]
LGTM

**Pull request merged**